application/manager/OrganisationManager.php

```diff
@@ -26,15 +26,61 @@ public function createOrganisation(string $nom, string $description, string $lie
             throw new Exception("error-creation-organisation-failed");
         }
         /** @var string $request2 */    
-        $request2 = "INSERT INTO `estAdmin` (`idUtilisateur`, `idOrganisation`) VALUES (
-            (SELECT idUtilisateur FROM Utilisateur WHERE nom = $userName), 
-            (SELECT idOrganisation FROM Organisation WHERE nom = $nom));";
+        $request2 = "INSERT INTO `estAdmin` (`id_Utilisateur`, `id_Organisation`) VALUES (
+            (SELECT id FROM Utilisateur WHERE nom = $userName), 
+            (SELECT id FROM Organisation WHERE nom = $nom));";
         $this->setQuery($request2);
         $this->query();
 
         return $this->ack("L'Organisation a bien été ajouté a la base de donnée");
     }
 
+    /**
+     * Cette fonction permet d'ajouter un membre dans une Organisation
+     * 
+     * @return array Cette fonction retourne ou un message d'erreur ou un message disant que tout c'est bien passer
+     * 
+     */
+    public function addUserToOrganisation(string $organisationId , string $userId): array {
+        /** @var string $request */
+        $request = "SELECT * FROM `Appartient`  WHERE id_Organisation = $organisationId AND id_Utilisateur = $userId ;";
+        $this->setQuery($request);
+        $result = $this->query();
+
+        if(count($result) >= 1){
+            throw new Exception("error-addMemberToOrganisation-failed");
```

Comment:
error-member-already-in-org

aussi c'est un check a faire dans le service, mais je m'en occuperais quand je ferai ma methode

Reply:
(donc c'est pas a modifier)